studio/artifact_store.py

```diff
@@ -1,6 +1,6 @@
-from firebase_artifact_store import FirebaseArtifactStore
-from gcloud_artifact_store import GCloudArtifactStore
-from s3_artifact_store import S3ArtifactStore
+from .firebase_artifact_store import FirebaseArtifactStore
```

Comment:
I think these are all Import issues resulting from path issues and not a Python3 vs Python2. Once things have been packaged up properly, should be okay though I've done something similar for development.